.travis.yml

```diff
@@ -19,6 +19,23 @@ env:
     - DEBUG_LINK=0 FASTFLASH=1
     - DEBUG_LINK=1 FASTFLASH=1
 
+matrix:
+  include:
+    - addons:
+        apt:
+          packages:
+            - gcc-multilib
+      env:
+        - EMULATOR=1 HEADLESS=1
+        - DEBUG_LINK=1
+      before_script:
+        - sed -i '/hidapi/d' "vendor/python-trezor/requirements.txt"
```

Comment:
python-trezor contains `--disable-hidapi` option in `setup.py`, not sure how we can utilize this when installing via pip, though

Reply:
@prusnak I tried that at least 5 different ways and gave up. :laughing: `pip` will not pass provided options to `./setup.py egg_info` which is what it uses to gather dependencies.

Reply:
Ok, glad to know I am the only one who failed on this. Thanks :-)

Reply:
@prusnak There is an issue here but I don't imagine it will be fixed https://github.com/pypa/pip/issues/4383